src/components/30-organisms/footer/index.js

```diff
@@ -37,6 +42,10 @@ class AXAFooter extends LitElement {
     `;
   }
 
+  static get resetHeadingCss() {
```

Comment:
Please add comment explaining why this seemingly unnecessary getter is infact needed!